ebml_matroska.xml

```diff
@@ -944,7 +944,7 @@
   <element name="TagName" path="1*1(\Segment\Tags\Tag\SimpleTag\TagName)" id="0x45A3" type="utf-8" minOccurs="1" maxOccurs="1" minver="1" webm="1">
     <documentation lang="en">The name of the Tag that is going to be stored.</documentation>
   </element>
-  <element name="TagLanguage" path="1*1(\Segment\Tags\Tag\SimpleTag\TagLanguage)" id="0x447A" type="string" minOccurs="1" maxOccurs="1" minver="1" webm="1" default="und">
+  <element name="TagLanguage" path="1*1(\Segment\Tags\Tag\SimpleTag\TagLanguage)" cppname="TagLangue" id="0x447A" type="string" minOccurs="1" maxOccurs="1" minver="1" webm="1" default="und">
```

Comment:
In a matter of sense, yes — it was a typo back when the original libMatroska implementation was written. The specs (even on the web page) came later and had to incorporate the existing class names.

Changing it now would break existing implementations.

We could consider using C++ type aliases for such cases; in C++11 syntax: `using KaxTagLanguage = KaxTagLangue` (or the other way around and rename the class), but I haven't fully thought about how that'll affect existing code, or if at all.

For now I'd just like to make the specs match existing source code out there.